lang/src/main/kotlin/org/partiql/lang/eval/builtins/MathFunctions.kt

```diff
@@ -29,6 +29,7 @@ import java.math.RoundingMode
 import kotlin.math.pow
 
 // TODO: this should be internal at the first place. Remove it in the next release.
+// Rather, un-deprecate and make internal in 0.10.0
```

Comment:
MathFunctions will be removed by https://github.com/partiql/partiql-lang-kotlin/pull/979